apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/Trace/SimpleManagedTracer.cs

```diff
@@ -145,6 +145,23 @@ public string GetCurrentTraceId()
             return _trace.TraceId;
```

Comment:
(this applies to line 62, but github won't let me comment on that line) "finised" should be "finished"